src/fifo/FifoBuffer.cpp

```diff
@@ -85,7 +85,7 @@ int32_t FifoBuffer::convertFramesToBytes(int32_t frames) {
 }
 
 int32_t FifoBuffer::read(void *buffer, int32_t numFrames) {
-    int32_t framesAvailable = mFifo->getFullFramesAvailable();
+    uint32_t framesAvailable = mFifo->getFullFramesAvailable();
```

Comment:
Please do not convert these to unsigned. See comments on https://github.com/google/oboe/pull/573